packages/example-apps/basic/src/utils/fn-helper.ts

```diff
@@ -0,0 +1,9 @@
+import { helper } from './helper-with-services';
```

Comment:
Why do you need `./helper-with-services` here? Seems like a simple function would work 🤔

Reply:
plain function not working for me :/

Reply:
do I need to use `setHelperManager` ?

Reply:
You should be able to import `fn` directly:

```
import { fn } from '@glimmer/helper';
```

It is implemented as a plain functoin.